source/DamageProfile.cpp

```diff
@@ -120,7 +127,6 @@ double DamageProfile::Scale(double scale, const Body &body, bool blast) const
 void DamageProfile::PopulateDamage(DamageDealt &damage, const Ship &ship) const
 {
 	const Outfit &attributes = ship.Attributes();
-	const Weapon &weapon = damage.GetWeapon();
```

Comment:
Removed this because this exact Weapon is already a variable of the class under the same name. It's redundant to grab it from the DamageDealt.